### PR TITLE
Issue 2767 - hzn secretmanager commands fail when there is a trailing…

### PR DESCRIPTION
**Pull request by MaxMcAdam**
… slash on the agbot url

### CODE REVIEW

File: exchange/secret.go [hunk 1]
```diff
@@ -29,10 +29,10 @@ func VaultSecretExists(ec ExchangeContext, agbotURL string, org string, userName
 	retryInterval := ec.GetHTTPFactory().GetRetryInterval()
 	for {
 		if err, tpErr := InvokeExchange(ec.GetHTTPFactory().NewHTTPClient(nil), "LIST", url, ec.GetExchangeId(), ec.GetExchangeToken(), nil, &resp); err != nil {
-			glog.Errorf(rpclogString(fmt.Sprintf(err.Error())))
+			glog.V(1).Infof(rpclogString(fmt.Sprintf(err.Error())))
```
Comment (dabooz) [inline exchange/secret.go:32]: Why did you change these?

**Review by dabooz — CHANGES_REQUESTED** [review]

File: exchange/secret.go (hunk 1 above)
Comment (MaxMcAdam) [reply exchange/secret.go:32]: These are called by `hzn deploycheck all` and cause this error message 
`ERROR: logging before flag.Parse: E0819 05:48:06.297389   24752 secret.go:32] Exchange RPC Unable to demarshal response "Permission denied, user \"maxwell.mcadam\" cannot list secret \"hw-secret-name\" in organization \"anaxsquad\"" from invocation of LIST at https://cp-console.roks-stg-local-70ea81cdef68a2eb78ece6d890b7dad3-0000.us-south.containers.appdomain.cloud/edge-agbot/org/anaxsquad/secrets/hw-secret-name, error: json: cannot unmarshal string into Go value of type exchange.VaultSecretExistsResponse`

**Review by dabooz — APPROVED** [review]
